src/x11/xf86WacomProperties.c

```diff
@@ -931,17 +931,22 @@ static int wcmSetProperty(DeviceIntPtr dev, Atom property, XIPropertyValuePtr pr
 			common->wcmPressureRecalibration = values[0];
 	} else if (property == prop_panscroll_threshold)
 	{
-		CARD32 *values = (CARD32*)prop->data;
+		INT32 *values = (INT32*)prop->data;
 
 		if (prop->size != 1 || prop->format != 32)
 			return BadValue;
 
-		if (values[0] <= 0)
-			return BadValue;
-
 		if (IsTouch(priv))
 			return BadMatch;
 
+		/* Reset to default if set to 0 */
+		if (values[0] == 0) {
+			values[0] = common->wcmResolY * 13 / 1000; /* 13mm */
+		}
+		if (values[0] == 0) {
+			values[0] = 1000;
+		}
```

Comment:
13 is a rather magic number to pick for this, any particular reason? I'm not disagreeing with it, but it would be good to know if there's more behind it than "seems good enough".

Reply:
Code was copied to match how the value is initialized on device connect. I probably should have refactored somehow, but laziness got the better of me :sweat_smile: The value 1000 was pretty much just a "right order of magnitude" choice that shouldn't normally be necessary (we "should" always have the kernel resolution) but avoids the possibility of division by zero.

https://github.com/linuxwacom/xf86-input-wacom/blob/742c3e4494bf89301031ba6a627a7f5d5f49556f/src/wcmCommon.c#L1565-L1572

Reply:
oh, indeed, the 13mm is documented in the man page. good enough for me, thanks!